packages/documentation/code/ts/src/Federated.lf

```diff
@@ -1,23 +1,25 @@
 target TypeScript {
-    // FIXME: This should work with timeout: 0 msec.
-    timeout: 1 msec
+  timeout: 1 msec  // FIXME: This should work with timeout: 0 msec.
```

Comment:
@edwardalee, I'm a little puzzled by this FIXME...

Reply:
I didn't add this comment, but I did notice that the other targets use a timeout of 0 for this test.